src/panels/lovelace/editor/config-elements/hui-area-card-editor.ts

```diff
@@ -64,6 +71,18 @@ export class HuiAreaCardEditor
           )}
           @value-changed=${this._valueChanged}
         ></ha-area-picker>
+        <ha-formfield
```

Comment:
Should we hide or disable the switch if there are no camera entities in the area?

Reply:
Don't think so, if the user later adds a camera to the area it should also work? 🤷‍♂️